iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/HelmInputFileContext.java

```diff
@@ -81,4 +84,8 @@ public boolean isInChartRootDirectory() {
     var rootChartDirectory = HelmFileSystem.retrieveHelmProjectFolder(Path.of(inputFile.uri()), sensorContext.fileSystem());
     return inputFile.path().getParent() != null && inputFile.path().getParent().equals(rootChartDirectory);
   }
+
+  public LinesShifting sourceMap() {
```

Comment:
I'm not sure if `sourceMap` is a good name for the method and field, as the type is `LinesShifting`. 
And in Java a `Map` usually means a Map as collection.